modules/dnn/test/test_ie_models.cpp

```diff
@@ -114,13 +114,21 @@ static const std::vector<std::string> getOpenVINOTestModelsList()
 
 static inline void genData(const std::vector<size_t>& dims, Mat& m, Blob::Ptr& dataPtr)
 {
+#if defined(INF_ENGINE_RELEASE) && INF_ENGINE_VER_MAJOR_LT(2019020000)
     std::vector<int> reversedDims(dims.begin(), dims.end());
     std::reverse(reversedDims.begin(), reversedDims.end());
 
     m.create(reversedDims, CV_32F);
     randu(m, -1, 1);
 
     dataPtr = make_shared_blob<float>(Precision::FP32, dims, (float*)m.data);
+#else
+    m.create(std::vector<int>(dims.begin(), dims.end()), CV_32F);
+    randu(m, -1, 1);
+
+    InferenceEngine::TensorDesc td(Precision::FP32, dims, Layout::ANY);
+    dataPtr = make_shared_blob<float>(td, (float*)m.data);
+#endif
```

Comment:
`TensorDesc::getDims ` returns reversed dims?

Reply:
No

Reply:
I mean there were
```cpp
std::vector<int> reversedDims(dims.begin(), dims.end());
    std::reverse(reversedDims.begin(), reversedDims.end());

    m.create(reversedDims, CV_32F);
    randu(m, -1, 1);
```
Shall we use std::reverse here or not? BTW, if 2018R5 and 2019R1 also support `make_shared_blob` from `TensorDesc` we can just rewrite it.